docs/getting-started/creating-a-database.mdx

```diff
@@ -0,0 +1,190 @@
+---
+sidebar_position: 1
+sidebar_label: "Create a Database"
+id: create-a-database
+title: Create a Database
+description: Creating and deploying a Kuneiform schema.
+slug: /getting-started/creating-a-database
+---
+
+In this tutorial, we will create a database using Kuneiform.  We will be deploying the database using the [Kwil CLI](../cli),
+however you can also deploy your database using the [Kuneiform IDE](<https://ide.kwil.com>).
+
+
+## Naming the Database
+
+Schemas are named using a database declaration at the top of the Kuneiform file.  Database names are unique to the account that deployed them.
+
+```sql title="social_media.kf"
+database social_media;
+```
+
+## Defining Tables
+
+Tables are defined using a table declaration.  Tables are named using table declarations at the top of the Kuneiform file.  Table names are unique to the database that contains them.
+
+```sql title="social_media.kf"
+// table for storing application users
+table users {
+    id int primary,
+    username text notnull unique minlen(5) maxlen(32),
+    age int notnull max(120),
+    address text notnull unique // user wallet addresses
+}
+
+// table for storing posts
+table posts {
+    id int primary,
+    title text notnull maxlen(300),
+    content text notnull maxlen(10000),
+    author_id int notnull,
+    post_date text notnull
+}
+```
+
+## Using Actions
+
+To define how users can interact with your database, you can define actions.  Actions contain SQL statements that are executed when an end user calls an action.
+
+Inputs are defined using a `$` symbol, followed by the variable name.  Contextual values, such as the wallet address calling your action, are specified using the `@` symbol, followed by the contextual value name.
```

Comment:
TODO: Add a link to the contextual values section, so people can see what contextual values are available.